src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mha.cpp

```diff
@@ -13,19 +13,21 @@ namespace ov {
 namespace test {
 namespace snippets {
 
+#define STATIC_SHAPE(...) {{}, {{__VA_ARGS__}}}
+
 namespace {
 
-const std::vector<std::vector<ov::PartialShape>> inputShapes_4D = {
-    {{1, 128, 12, 64}, {1, 128, 12, 64}, {1, 12, 128, 128}, {1, 128, 12, 64}},
-    {{1, 128, 16, 64}, {1, 128, 16, 64}, {1, 16, 1, 1}, {1, 128, 16, 64}},
-    {{1, 128, 16, 64}, {1, 128, 16, 64}, {1, 1, 1, 128}, {1, 128, 16, 64}},
-    {{2, 68, 6, 92}, {2, 68, 6, 92}, {1, 1, 68, 68}, {2, 68, 6, 92}},
-    {{1, 58, 16, 34}, {1, 58, 16, 34}, {1, 1, 1, 58}, {1, 58, 16, 34}},
+const std::vector<std::vector<ov::test::InputShape>> inputShapes_4D = {
```

Comment:
By the way, for static tests you can use `ov::Shape` and call `ov::test::static_shapes_to_test_representation(inputShapes_4D)` in test instances ([example](https://github.com/openvinotoolkit/openvino/blob/36b62da080ca3334c0270c607c2b62e7850621fd/src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/adaptive_pooling.cpp#L25)).
Then you don't have to define macro `STATIC_SHAPE`

Reply:
Why will you have to define macro again?
On the contrary: you don't have to write this macro everywhere or use a lot of parentheses. You just define static shapes:
```
const std::vector<std::vector<ov::Shape>> static_inputShapes_4D = {  // instead of ov::test::InputShape
    {{1, 128, 12, 64}, {1, 128, 12, 64}, {1, 12, 128, 128}, {1, 128, 12, 64}},
    {{1, 128, 16, 64}, {1, 128, 16, 64}, {1, 16, 1, 1}, {1, 128, 16, 64}},
    {{1, 128, 16, 64}, {1, 128, 16, 64}, {1, 1, 1, 128}, {1, 128, 16, 64}},
    {{2, 68, 6, 92}, {2, 68, 6, 92}, {1, 1, 68, 68}, {2, 68, 6, 92}},
    {{1, 58, 16, 34}, {1, 58, 16, 34}, {1, 1, 1, 58}, {1, 58, 16, 34}},
};
```
And then in instance just use `ov::test::static_shapes_to_test_representation(static_inputShapes_4D )`:
```
INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MHA_4D, MHA,
::testing::Combine(::testing::ValuesIn(ov::test::static_shapes_to_test_representation(static_inputShapes_4D)),
....,
MHA::getTestCaseName);
```
I think it's really better than to write `STATIC_SHAPE` everywhere :grin:

Reply:
Macro is needed to avoid writing `ov::test::static_shapes_to_test_representation` every time.
Ok, honestly I don't think that it's that important, so I deleted the `STATIC_SHAPE` macro, since it bothers you so much 🙂

Reply:
> since it bothers you so much
When I see many-many-many `STATIC_SHAPE` in the code, I'm confused.
Moreover, test infrastructure already implemented the helper for these case.
Why not use what has already been implemented? 😃 
Now, the code is clearer and better than before.
Thanks!